messages/commands/eth.proto

```diff
@@ -0,0 +1,68 @@
+// Copyright 2019 Shift Cryptosecurity AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+// TODO(nc): This should go into the main proto file in a future refactor
+message PubResponse {
```

Comment:
hmm what about putting inti common.proto and including it here and in the file with the btc messages?

Reply:
Interesting suggestion. I'll try it.

Reply:
i'll have one more of those with the nonce covert channel fix, which will be used for in btc and eth, so it would be nice if it worked :)

Reply:
It worked perfectly